### PR TITLE
Fix bug in AskForValueDialog in typeAndWaitText method

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Fix bug in AskForValueDialog in typeAndWaitText method

### What issues does this PR fix or reference?
Issue https://github.com/eclipse/che/issues/7543

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java [hunk 1]
```diff
@@ -137,7 +138,10 @@ public void clickCancelButtonJava() {
   }
 
   public void typeAndWaitText(String text) {
-    input.sendKeys(text);
+    waitFormToOpen();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java]: It should be checked before the call of the method.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java [hunk 2]
```diff
@@ -137,7 +138,10 @@ public void clickCancelButtonJava() {
...
+    new WebDriverWait(seleniumWebDriver, TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java]: Good catch!
Lets extract `new WebDriverWait(seleniumWebDriver, TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC)` into the separate constant and add similar verification in other methods.

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java [hunk 3]
```diff
@@ -105,90 +108,88 @@ public AskForValueDialog(
...
+    loadPageWait.until(visibilityOf(okBtnnewJavaClass)).click();
     loader.waitOnClosed();
   }
 
   public void typeTextInFieldName(String text) {
-    inputNameJavaClass.sendKeys(text);
+    loadPageWait.until(visibilityOf(inputNameJavaClass)).sendKeys(text);
+  }
+
+  public void waitTextInFieldName(String expectedText) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java]: IMHO it's better to enclose call of this method into the **typeTextInFieldName()** in similar way, as it was done for **waitInputNameContains()** method.

File: selenium/che-selenium-test/src/test/resources/conf/selenium.properties [hunk 4]
```diff
@@ -31,11 +31,11 @@ che.selenium.infrastructure=docker
 # Define configuration for che default test user.
 # For che single user assembly, the default values will be enough.
 # For che multiuser assembly, user must be already created on identity provider service.
-che.test_user.email=che@eclipse.org
-che.test_user.password=secret
+che.test_user.email=admin@admin.com
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/conf/selenium.properties]: Is it really what you wanted to change? :-)

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**